control/installation.xml

```diff
@@ -14,7 +14,9 @@ textdomain="control"
         <label>GNOME Desktop (Basic)</label>
       </basic_desktop_host_role>
       <basic_desktop_host_role_description>
-        <label>GNOME Desktop environment with only essential graphical applications installed (File Manager, Web Browser) and using X11 technology</label>
+	      <label>• Basic GNOME Desktop environment.
```

Comment:
You used 4 spaces instead of 2 to indent the line :)

Reply:
Do we care? It's a French standard, most probably :)